marconi-chain-index/test/Spec/Golden/Cli/marconi-sidechain___help.help

```diff
@@ -0,0 +1,31 @@
+marconi-sidechain - a lightweight customizable solution for indexing and
```

Comment:
Why were the following two files added? They are for marconi-sidechain, not marconi-chain-index

Reply:
I probably run the test in the wrong directory and didn't notice it on git add, I'm binning them.